### PR TITLE
1.x

### PR DESCRIPTION
**Pull request by arshadkhan53**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
+{
+  "name": "drupal/ezcontent_publish",
+  "description": "A Feature rich stater profile from Srijan.",
```
Comment (vaibhavjain-in) [inline composer.json]: A feature rich module enhancing the experience of Publishers.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        "type": "drupal-library",
+        "dist": {
+          "url": "https://download.ckeditor.com/layoutmanager/releases/layoutmanager_1.1.0.zip",
+          "type": "zip"
+        }
+      }
+    }
+
+
+
```
Comment (vaibhavjain-in) [inline composer.json]: Lets remove these lines.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Comment by arshadkhan53** [comment]
We would be removing master branch, hence closing this pr.

**Pull request closed without merging**